packages/apputils-extension/src/index.ts

```diff
@@ -606,6 +615,17 @@ const utilityCommands: JupyterFrontEndPlugin<void> = {
         }
       }
     });
+
+    commands.addCommand(CommandIDs.displayShortcuts, {
+      label: trans.__('Display Keyboard Shortcuts'),
+      caption: trans.__(
+        'Display relevant keyboard shortcuts for the current active element'
+      ),
+      execute: args => displayShortcuts(app, trans)
+    });
+
+    const category: string = trans.__('Help');
+    palette.addItem({ command: CommandIDs.displayShortcuts, category });
```

Comment:
As `palette` is optional:

```suggestion
    if (palette) {
      const category: string = trans.__('Help');
      palette.addItem({ command: CommandIDs.displayShortcuts, category });
    }
```